src/styles/_header.scss

```diff
@@ -38,4 +39,21 @@
     font-size: 12px;
     font-weight: 500;
   }
+
+  &__item:hover {
+    color: #34568b;
+    transform: rotate(5deg) translateY(-8px);
+  }
+
+  &__item:hover::after {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    content: "";
+    background-color: #00acdc;
```

Comment:
:hover should be applied only to the card, do not add it to the title directly
![Screenshot 2022-07-22 at 08 51 20](https://user-images.githubusercontent.com/17098736/180372128-b919922a-5d5e-4c8a-9b1d-590cdfac4522.png)